htdocs/compta/stats/casoc.php

```diff
@@ -458,8 +467,19 @@
 if ($subcat) {
 	print ' checked';
 }
-print'></td>';
-print '<td colspan="7" class="right">';
+
+print'>';
+// type filter (produit/service)
+print ' ';
+print '&nbsp;&nbsp;';
+print $langs->trans("Type").': ';
```

Comment:
In look and feel of recent version, labels of filters must be included as the placeholder of the combobox (to move into the param$shwo_empty of selectarray